### ISSUE TITLE
Typing mismatch for `Handsontable.plugins.ContextMenu`

### ISSUE THREAD

**Issue by briwa** [handsontable/handsontable#6347]
### Description
We are using a custom context menu for our Handsontable. We'd like to retrieve values such as the context menu separator and default items.

![Screen Shot 2019-10-16 at 11 49 05 AM](https://user-images.githubusercontent.com/8046636/66886734-fe66cd00-f00a-11e9-920c-c9b5be3068ee.png)

However, in TS, it is not there.

![Screen Shot 2019-10-16 at 11 49 59 AM](https://user-images.githubusercontent.com/8046636/66886766-1a6a6e80-f00b-11e9-844b-fa3d9e63dc11.png)

In the definition file, it is defined as [something else](https://github.com/handsontable/handsontable/blob/6.2.2/handsontable.d.ts#L755-L764) which is different than what it is being printed out in the console.

If this was meant to be private, where else should we retrieve this from? I went through [the definition file](https://github.com/handsontable/handsontable/blob/6.2.2/handsontable.d.ts) but I couldn't find a good place to retrieve it from. Thank you for the help.

### Steps to reproduce

1. Get `Handsontable.plugins.ContextMenu.DEFAULT_ITEMS` in TS file
2. It's not there

### Demo
https://codesandbox.io/s/handsontable-typing-bug-2r45j
Head on to `index.ts` and you'll see that the values are printed in the browser, but it's not there on TS.

### Your environment
* Handsontable version: 6.2.2
* Browser Name and version: Any browser
* Operating System: Any OS

**Comment by AMBudnik** [comment]
Hey @briwa 

that's a static and statics seem to be private. 
![Zrzut ekranu 2019-10-18 o 07 48 27](https://user-images.githubusercontent.com/10757813/67068943-dcd92300-f17b-11e9-9669-e9ee6f9e40f1.png)

However, I can mark this topic as a feature request.

**Comment by briwa** [comment]
@AMBudnik Thank you!

**Comment by AMBudnik** [comment]
Hi @briwa ,

I have good news! We've just released 8.0.0-beta2 that fixes the mentioned issue. We won't be closing this ticket now. It will be closed after we release the final 8.0.0 version of Handsontable.

Here’s a full list of changes https://github.com/handsontable/handsontable/releases/tag/8.0.0-beta.2
NPM https://www.npmjs.com/package/handsontable/v/8.0.0-beta.2 

All feedback's appreciated. Thank you for your input.

**Comment by briwa** [comment]
@AMBudnik Thank you for looking into this. I'll check it out. Closing this issue then.

**Comment by AMBudnik** [comment]
@briwa We will be closed after we release the final 8.0.0 version of Handsontable.

**Comment by aninde** [comment]
The lines below are present in `handsontable.d.ts` `pre-8.0.0-rev1`.   
```
 interface ContextMenu extends Base {
      DEFAULT_ITEMS: contextMenu.PredefinedMenuItemKey[];
      SEPARATOR: SeparatorObject;
```

After updating handsontable-react wrappers to version 8.0.0. it should work.

**Comment by AMBudnik** [comment]
Hi @briwa we can call this issue solved in v8.0.0.

Here are the CDN resources https://cdn.jsdelivr.net/gh/handsontable/handsontable@latest/dist/handsontable.full.css
https://cdn.jsdelivr.net/gh/handsontable/handsontable@latest/dist/handsontable.full.js
NPM npm i handsontable
Release Notes https://github.com/handsontable/handsontable/releases

I hope that it works for you as well. Feel free to share any feedback.
